manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.Extensions</ProviderNameSpace>
```

Comment:
@boumenot we can use the same subscription as `Microsoft.Azure.Extensions.LinuxAsm` or `Microsoft.Azure.Extensions.TestLAD`. Are they already in EDP?